src/pages/app.ts

```diff
@@ -1,5 +1,5 @@
-const express = require("express");
-const path = require("path");
+import express = require("express");
+import path = require("path");
 
 const app = express();
 const PORT = 3000;
```

Comment:
Порт часто задается из переменной окружения, поэтому тут лучше написать так
```suggestion
const PORT = process.env.port || 3000;
```
Сейчас это может казаться бессмысленным, но это потому что ты запускаешь в основном локально на одной машине, а когда приложения будет разворачиваться многими людьми в докере на разных машинах, это может сильно пригодиться. Чтобы задать порт через переменную окружения, можно либо запускать скрипт так 
```
PORT=3000 node app.js
```

Либо создать папку .env и там прописать PORT=3000